### ISSUE TITLE
Implementing multi-threading in code.

### ISSUE THREAD

**Issue by rahulnegi20** [rahulnegi20/tdict#2]
If we can use multi-threading to call for synonyms and examples while waiting for or `get-meaning` function to respond.

**Comment by SineshX** [comment]
yes I can try , but first fix the get_meaning and get example function : its showing error  :---- 

!!Error!!  <class 'IndexError'>

**Comment by rahulnegi20** [comment]
sure will look into it.
